contracts/AelinUpFrontDeal.sol

```diff
@@ -578,14 +602,6 @@ contract AelinUpFrontDeal is AelinERC20, MinimalProxyFactory, IAelinUpFrontDeal
         return (poolSharesPerUser[_address]);
     }
 
-    /**
-     * @dev getAmountVested
-     * @param _address address to check
-     */
-    function getAmountVested(address _address) public view returns (uint256) {
```

Comment:
Why are we using all these getter functions at the end since the variable are all public?

Reply:
we can get rid of those